lua/org-bullets.lua

```diff
@@ -35,6 +45,11 @@ local function set_config(conf)
   if conf.symbols and type(conf.symbols) == "function" then
     conf.symbols = conf.symbols(symbols) or symbols
   end
+
+  if conf.bullet_chars and type(conf.bullet_chars) == "function" then
```

Comment:
Only the type check is necessary here, since if bullet chars were nil it wouldn't pass the check anyway